lib/rspec/rails/matchers/have_http_status.rb

```diff
@@ -233,6 +233,7 @@ def set_expected_code!
         #   expect(response).to have_http_status(:error)
         #   expect(response).to have_http_status(:missing)
         #   expect(response).to have_http_status(:redirect)
+        #   expect(response).to have_http_status(:unauthorized)   
```

Comment:
There is trailing whitespace on this line which is why the build is failing.

Reply:
Ok. Will fix - I didn't know that a comment would fail the test.

Reply:
Its not the comment that fails the test, its the whitespace, we have linters :)